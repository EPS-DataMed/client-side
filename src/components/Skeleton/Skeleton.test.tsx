import { render } from '@testing-library/react'
import 'jest-styled-components' // para asserções de estilos
import { Skeleton } from '.'

describe('Skeleton', () => {
  it('renders correctly', () => {
    const { container } = render(<Skeleton />)
    expect(container.firstChild).toMatchSnapshot()
  })

  it('has correct animation styles', () => {
    const { container } = render(<Skeleton />)
    const divElement = container.firstChild
    expect(divElement).toHaveStyleRule(
      'animation',
      expect.stringContaining('1s linear infinite alternate'),
    )
  })
})
