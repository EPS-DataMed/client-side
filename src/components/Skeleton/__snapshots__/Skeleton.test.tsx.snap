// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton renders correctly 1`] = `
.c0 {
  animation: gulyfb 1s linear infinite alternate;
}

<div
  class="c0"
/>
`;
